run a minimal server (#954)
jamulussoftware/jamulussoftware.github.io: wiki/en/Running-a-Server.md

---

### Minimum setup to run in Registered mode

> You can run headless Jamulus Server on Linux using `systemctl`, which is covered in other sections.

Reviewer: ```suggestion
You can run a headless Server on Linux using systemd (with `systemctl`), which is covered in other sections.
```
Per style guide we omit Jamulus if not absolutely necessary.

> You can run headless Jamulus Server on Linux using `systemctl`, which is covered in upcoming sections.

Author: ```suggestion
You can run headless Jamulus Server on Linux using `systemctl`, which is covered in other sections.
```

> You can also run Jamulus Server from the command line. Enter the following command to [run a Registered Server](Running-a-Server#server-types):

Reviewer: ```suggestion
You can also run a Server directly from the command line. Enter the following command to [run a Registered Server](Running-a-Server#server-types):
```

~~~
jamulus-headless --nogui --server \
        --directoryaddress hostname:port \

>         --serverinfo "yourServerName;yourCity;[country ID]"

Reviewer: ```suggestion
        --serverinfo "[name];[city];[country as two-letter ISO country code or Qt5 Locale]"
```

~~~

See the table below for `hostname:port` values.

> **Note**: Semicolon and newline characters are not allowed in `yourServerName` and `yourCity` within the `--serverinfo` argument. 

Reviewer: ```suggestion
**Note**: Semicolon and newline characters are not allowed in `[name]` and `[ciry]` within the `--serverinfo` argument. See the [Command line options](#command-line-options) for more information on the country. 
```

The link to command line options will need to be checked, but the linkchecker will later on complain.

Author: Why would this local anchor work? I don't see a command-line-options keyword, and I don't see a title where these dashes are spaces. This text links to Running-a-Server#configuration-options elsewhere in the file. But that's the section we're in. And [that section](https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Running-a-Server.md?plain=1#L166-L177) does not mention Qt5 Locale codes. Where are those? I'd rather duplicate an offsite link here because local anchors often just confuse people.

Reviewer: The linkchecker will complain and then we'd find the correct section. It should be hidden behind an import in the site and only shows up in compiled form.

> **Note**: Semicolon and newline characters are not allowed in `[name]` and `[ciry]` within the `--serverinfo` argument. See the [Command line options](#command-line-options) for more information on the country. 

Author: ```suggestion
**Note**: Semicolon and newline characters are not allowed in `[name]` and `[city]` within the `--serverinfo` argument. See the [Command line options](#command-line-options) for more information on the country. 
```

> **Note**: Semicolon and newline characters are not allowed in `[name]` and `[city]` within the `--serverinfo` argument. See the [Command line options](#command-line-options) for more information on the country. 

Reviewer: ```suggestion
**Note**: Semicolon and newline characters are not allowed in `[name]` and `[city]` within the `--serverinfo` argument. See the [Command line options](#command-line-options) for more information on the country code. 
```
